pytroll_collectors/tests/test_triggers.py

```diff
@@ -264,3 +264,48 @@ def test_getting_metadata_does_not_involve_multiple_sections(self):
                        'uri': 'somefile_20220512T1544_20220512T1545.data',
                        'key1': "value1"
                        }
+
+    def test_filetrigger_exception(self, caplog):
+        """Test getting the metadata."""
+        from pytroll_collectors.triggers._base import FileTrigger
+        import configparser
+
+        def _collectors(metadata):
+            raise KeyError("Found no TLE entry for 'METOP-B' to similate KeyError")
+        collectors = [_collectors]
+        config = configparser.ConfigParser(interpolation=None)
+        section = "section1"
+        config.add_section(section)
+        config.set(section, "pattern", "{name}_{start_time:%Y%m%dT%H%M}_{end_time:%Y%m%dT%H%M}.data")
+        publisher = None
+        trigger = FileTrigger(collectors, dict(config.items("section1")), publisher, publish_topic=None,
+                              publish_message_after_each_reception=False)
+        trigger._process_metadata({'sensor': 'avhrr', 'platform_name': 'Metop-B',
+                                   'start_time': datetime(2022, 9, 1, 10, 22, 3),
+                                   'orbit_number': '51653',
+                                   'uri': 'AVHRR_C_EUMP_20220901102203_51653_eps_o_amv_l2d.bin',
+                                   'uid': 'AVHRR_C_EUMP_20220901102203_51653_eps_o_amv_l2d.bin',
+                                   'origin': '157.249.16.188:9062',
+                                   'end_time': datetime(2022, 9, 1, 10, 25, 3)})
+        assert "Found no TLE entry for 'METOP-B' to similate KeyError" in caplog.text
```

Comment:
```suggestion
        assert "Found no TLE entry for 'METOP-B' to simulate" in caplog.text
```

Reply:
Hmm, where does this exception text originate? I found `"Found no TLE entry for '%s'"` in https://github.com/pytroll/pyorbital/blob/main/pyorbital/tlefile.py#L200 but the rest of it I can't seem to find in either Pyorbital, Pytroll Schedule nor Pytroll Collectors.

Reply:
oh, how to explain

I just added some dummy text in the test to be sure I matched the test KeyError message and not the message from the actual code. If that make any sense. I struggled a bit before I got this right.

But sure, I can at least get the spelling right.

Reply:
Oh, right. Didn't grep the same file :sweat_smile: